src/components/LoadingButton/Container.story.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs/react';
+
+import withTests from '../../util/withTests';
+import LoadingButton from '.';
+
+storiesOf('LoadingButton', module)
+  .addDecorator(withTests('LoadingButton'))
+  .add(
+    'LoadingButton with Success animation',
+    withInfo()(() => (
+      <LoadingButton
+        isLoading={boolean('Loading', false)}
+        onClick={action('clicked')}
+        onAnimationComplete={action('animation completed')}
+        exitAnimation={LoadingButton.SUCCESS}
+        primary
+      >
+        Touch my knob
```

Comment:
Say what 😅

Reply:
Ah! It's to alert about the storybook knob. Does this sound bad?